tests/flow/test_v7_encode_decode.py

```diff
@@ -128,3 +128,28 @@ def test07_index_after_encode_decode_in_v7(self):
         plan = redis_graph.execution_plan(
             "MATCH (n:N {val:1}) RETURN n")
         self.env.assertIn("Index Scan", plan)
+
+    def test08_multiple_graphs_with_index(self):
+        # Create a multi-key graph.
+        graph1_name = "v7_graph_1"
+        graph1 = Graph(graph1_name, redis_con)
+        graph1.query("UNWIND range(0,21) AS i CREATE (a:L {v: i})-[:E]->(b:L2 {v: i})")
```

Comment:
I would define the `VKEY` size as a global variable and use it here with some factor